src/sentry/static/sentry/app/components/panels/panel.tsx

```diff
@@ -33,6 +33,7 @@ const Panel = styled(({title, body, dashedBorder, ...props}: PanelProps) => {
   box-shadow: ${p => (p.dashedBorder ? 'none' : p.theme.dropShadowLight)};
   margin-bottom: ${space(3)};
   position: relative;
+  overflow: hidden;
```

Comment:
I think this may have unintended consequences for our existing Panels (dropdowns/tooltips) that won't show up in our percy tests. I would err on the side of caution and extend this Panel for this grid component.